packages/p2p-v2/src/pages/orders/screens/Orders/OrdersTable/OrdersTable.scss

```diff
@@ -4,8 +4,15 @@
             grid-template-columns: 1fr 1.5fr 2fr 3fr 1.5fr 1.5fr 1.5fr;
         }
 
-        &__content__row {
-            border-bottom: 1px solid #f2f3f4;
+        &__content {
+            @include mobile {
+                // stylelint-disable-next-line declaration-no-important
+                height: calc(100vh - 19.5rem) !important;
+            }
```

Comment:
👀 
```suggestion
            @include mobile {
                // stylelint-disable-next-line declaration-no-important
                height: calc(100vh - 19.5rem) !important;
            }
```

Reply:
need to have this cause the order table on mobile isnt scrollable 😓 and I need to override it

Reply:
@ameerul-deriv  the !important is not needed. it works without it. can you check again ?